src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {` `}
+      {person.name}
```

Comment:
```suggestion
      {`My name is ${person.name}`}
```
use template string here